src/tools/rbd_mirror/InstanceReplayer.cc

```diff
@@ -649,10 +650,30 @@ void InstanceReplayer<I>::start_group_replayer(
   ceph_assert(ceph_mutex_is_locked(m_lock));
 
   std::string global_group_id = group_replayer->get_global_group_id();
+  std::string group_name = group_replayer->get_name();
+  std::string group_id;
+  bool resync_requested = false;
+  int r = librbd::cls_client::mirror_group_resync_get(&m_local_io_ctx,
+                                                      global_group_id,
+                                                      group_name,
+                                                      &group_id);
+  if (r < 0) {
+    derr << "getting mirror group resync for global_group_id="
```

Comment:
This code should be moved to the GroupReplayer. But just adding a comment to point out that this error message should only be printed when ( r != -ENOENT) or it will flood the logs.